freedomofpress/securedrop-docs
# Intel now provides SHA1 checksums for BIOS downloads

## Status


Ready for review

## Description of Changes

Swap all the language referring to MD5 hashes to SHA1 instead, and
update the screenshot accordingly.

Fixes #336.

## Testing
* Verify Intel now provides SHA1
* Verify screenshot matches Tails UI

## Release 

* n/a


## Checklist (Optional)

- [ ] Doc link linting (`make docs-linkcheck`) passed
